frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeSidebar.tsx

```diff
@@ -25,20 +25,11 @@ export function SummarizeSidebar({
   onQueryChange,
   onClose,
 }: SummarizeSidebarProps) {
-  const {
-    query,
-    stageIndex,
-    aggregations,
-    hasAggregations,
-    handleQueryChange,
-    handleAddBreakout,
-    handleUpdateBreakout,
-    handleRemoveBreakout,
-    handleReplaceBreakouts,
-  } = useSummarizeQuery({
-    query: initialQuery,
-    onQueryChange,
-  });
+  const { query, handleUpdateQuery, handleAggregationChange, hasAggregations } =
```

Comment:
This contains the default query behavior that we want for the query builder view when using the `SummarizeSidebar`. When the user removes all of the aggregations, this hook adds the `Count` aggregation when the user exits and re-enters the `SummarizeSidebar` as a sane default. It also prevents the query builder from reloading the visualization if the user removes the default `Count` aggregation since interally, nothing has changed until the user has actually added a breakout or another aggregation